scripts/change-log-generator.js

```diff
@@ -0,0 +1,249 @@
+/*
+ * Automates the process of building the change log for releases.
+ *
+ * Assumptions:
+ * 1. You have shelljs installed globally using `npm install -g shelljs`.
+ * 2. The release branch in question has already been cut.
+ * 3. readline-sync is installed using 'npm install readline-sync'
+ *
+ * This does not currently take into consideration:
+ * 1. External contributions
+ * 2. Duplicates (ex file changes made in both salesforcedx-apex-debugger and salesforcedx-apex-replay-debugger)
+ * 3. Non-salesforce package contributions aside from doc updates
+ * 4. Adding vs. Fixed vs. Ignore in change log
+ */
+
+const process = require('process');
+const path = require('path');
+const shell = require('shelljs');
+const readline = require('readline-sync');
+const fs = require('fs');
+const util = require('util');
+
+// Constants
+const CHANGE_LOG_PATH = path.join(
+  process.cwd(),
+  'packages',
+  'salesforcedx-vscode',
+  'CHANGELOG.md'
+);
+const CHANGE_LOG_BRANCH = 'changeLog-v';
+
+// Change log values
+const LOG_HEADER =
+  '%s - (INSERT RELEASE DATE [Month Day, Year])\n' +
```

Comment:
Since we typically do releases weekly on Thursdays, would it be possible to have this be an auto-generated value that we can manually change if needed?

Reply:
Yes, I think so. I'm glad you mentioned it. I wasn't sure if that was something that people would think was overkill. I'll look into adding it.